YMChat/YMConfig.swift

```diff
@@ -57,6 +58,7 @@ open class YMConfig: NSObject {
             queryItems.append(URLQueryItem(name: "ym.payload", value: decodedPayload))
         }
         queryItems.append(URLQueryItem(name: "version", value: "\(version)"))
+        queryItems.append(URLQueryItem(name: "disableActionsOnLoad", value: String(disableActionsOnLoad)))
```

Comment:
Please use the string conversion syntax as present on previous line. Or change it there. Just keep it uniform